apps/language_server/lib/language_server/providers/document_symbols.ex

```diff
@@ -148,6 +151,18 @@ defmodule ElixirLS.LanguageServer.Providers.DocumentSymbols do
     }
   end
 
+  # @impl true
+  defp extract_symbol(_current_module, {:@, _, [{:impl, location, [true]}]}) do
+    %{type: :constant, name: "@impl true", location: location, children: []}
+  end
+
+  # @impl BehaviourModule
+  defp extract_symbol(_current_module, {:@, _, [{:impl, location, [impl_expression]}]}) do
```

Comment:
Let's do the same for `@behaviour`

Reply:
ah, good idea! I've added that to the PR.